pages/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Repair damaged or corrupted `pcap` and `pcapng` files.
+> More information: <https://f00l.de/pcapfix/>.
+
+- Repair a `pcap`/`pcapng` file:
+
+`pcapfix {{path/to/file.pcap}}`
+
+- Repair an entire `pcap`/`pcapng` file:
+
+`pcapfix --deep-scan {{path/to/file.pcap}}`
+
+- Repair a `pcap`/`pcapng` file and write the repaired file to the specified location:
+
+`pcapfix --outfile {{path/to/repaired.pcap}} {{path/to/file.pcap}}`
+
+- Repair a `pcapng` file and treat it as a `pcapng` file, ignoring the automatic recognition:
```

Comment:
```suggestion
- Repair a `pcap` file and treat it as a `pcapng` file, ignoring the automatic recognition:
```
Shouldn't this be repair a `pcap` file here?
Or is the placeholder below wrong?

Reply:
The placeholder is incorrect. Thanks for the hint!